integration/tests-common/integration-test-utils/src/main/java/org/wso2/esb/integration/common/utils/clients/axis2client/AxisServiceClient.java

```diff
@@ -46,7 +46,13 @@ public OMElement sendReceive(OMElement payload, String endPointReference, String
             options = new Options();
             options.setTo(new EndpointReference(endPointReference));
             options.setProperty(org.apache.axis2.transport.http.HTTPConstants.CHUNKED, Boolean.FALSE);
-            options.setTimeOutInMilliSeconds(45000);
+            //Increase ReadTimeOut to 3 mins in Windows environments since the operation execution is taking longer in
+            // Windows
+            if (System.getProperty("os.name").contains("Windows")) {
```

Comment:
Better to make it , wdyt ?
```suggestion
            if (System.getProperty("os.name").toLowerCase().contains("windows")) {
```